### PR TITLE
Update DATS.json

### PR DESCRIPTION
**Pull request by GHPBZ**
Update to Creators.

### CODE REVIEW

File: DATS.json [hunk 1]
```diff
@@ -481,34 +507,26 @@
...
-	{
       "category": "registrationPage",
       "values": [
         {
           "value": "http://www.cima-q.ca/en/data-cima-q/"
         }
       ]
     },
-	{
-      "category": "downloadOptions",
```
Comment (laemtl) [inline DATS.json:500]: Is this change intended?

**Comment by GHPBZ** [comment]
No, I don't think so because I only changed the PI/Creators section in this pass.  However, I was working from a version that a CIMA-Q coordinator gave me with updates, so perhaps those fields were deleted by the DATS editor that doesn't yet handle the downloadOptions.  In any case, no that shouldn't be deleted.  Well spotted.

**Review by laemtl — APPROVED** [review]
